DeskReserve/appsettings.json

```diff
@@ -6,7 +6,7 @@
     }
   },
   "ConnectionStrings": {
-    "DefaultConnection": "Server=myServerAddress;Database=myDataBase;User Id=myUsername;Password=myPassword;"
+    "DefaultConnection": "Server=yonchopc;Database=test123;Trusted_Connection=True;"
```

Comment:
Why edit the existing connection string

Reply:
Because i needed to connect the application with my localhost to test the functionality.